cabinets/parser/txt.py

```diff
@@ -0,0 +1,13 @@
+from cabinets.parser import register_extensions, Parser
+
+
+@register_extensions('txt')
+class PlainTextParser(Parser):
+
+    @classmethod
+    def load_content(cls, content):
+        return content.decode()
+
+    @classmethod
+    def dump_content(cls, data, encoding='utf-8'):
```

Comment:
The interesting thing about using a kwarg here is that there is no way to actually access it from standard `cabinets` methods (`cabinets.read()`, `cabinets.create()`). This brings up an interesting question of whether we need to allow for some configuration for parsers to be passed with the top level `cabinets` methods.

Reply:
Yup, sounds like a problem that needs more thought. I'm just gonna default to `utf-8` and we should reevaluate in a new PR.